### PR TITLE
Bump max wasm file size to 2MB

### PR DESCRIPTION
**Pull request by LCyson**
## Describe your changes and provide context
This change bumps the maximal uploadable wasm file size from 1024 * 1024 to 1280 * 1024.

## Testing performed to validate your change
- unit test
- e2e contract deployment test

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -1578,5 +1578,5 @@ func (app *App) decorateContextWithDexMemState(base context.Context) context.Con
 
 func init() {
 	// override max wasm size to 1MB
-	wasmtypes.MaxWasmSize = 1024 * 1024
+	wasmtypes.MaxWasmSize = 1280 * 1024
```
Comment (codchen) [inline app/app.go]: any particular reason to choose this number vs. maybe just 2MB? also the comment above needs to be updated

**Review by codchen — APPROVED** [review]

File: app/app.go (hunk 1 above)
Comment (philipsu522) [reply app/app.go]: +1

**Review by philipsu522 — APPROVED** [review]

File: app/app.go (hunk 1 above)
Comment (LCyson) [reply app/app.go]: I'm inclined not to bump the uploadable wasm size too much as larger wasm file has negative impact on chain performance. Choosing this number to unblock Levana. We can also make a chain upgrade to adjust the param here, but let me know if you have a different thought

Comment (LCyson) [reply app/app.go]: will bump it to 2MB for now

**Pull request merged**